client/components/summary/placeholder.js

```diff
@@ -3,24 +3,35 @@
  * External dependencies
  */
 import { Component } from '@wordpress/element';
+import classnames from 'classnames';
 import { range } from 'lodash';
 import PropTypes from 'prop-types';
 
 /**
  * Internal dependencies
  */
-import SummaryList from './index';
+import { isMobileViewport, isTabletViewport } from 'lib/ui';
 
 /**
  * `SummaryListPlaceholder` behaves like `SummaryList` but displays placeholder summary items instead of data.
- * This can be used while loading.
+ * This can be used while loading data.
  */
 class SummaryListPlaceholder extends Component {
 	render() {
-		const { numberOfItems } = this.props;
+		const isDropdownBreakpoint = isTabletViewport() || isMobileViewport();
+		const numberOfItems = isDropdownBreakpoint ? 1 : this.props.numberOfItems;
+
+		const hasItemsClass = numberOfItems < 10 ? `has-${ numberOfItems }-items` : 'has-10-items';
+		const classes = classnames( 'woocommerce-summary', {
+			[ hasItemsClass ]: ! isDropdownBreakpoint,
+		} );
+		const containerClasses = classnames( 'woocommerce-summary__item-container', {
```

Comment:
You can remove this variable & just add the className inline, since it's not actually conditional